src/ml/neural_net/tf_compute_context.cpp

```diff
@@ -269,5 +268,56 @@ tf_model_backend::~tf_model_backend() {
   call_pybind_function([&]() { model_ = pybind11::object(); });
 }
 
+tf_image_augmenter::tf_image_augmenter(const options& opts)
+    : processed_image_augmenter(opts) {}
+
+image_augmenter::result tf_image_augmenter::prepare_images(
+    std::vector<labeled_image> source_batch) {
+  return processed_image_augmenter::prepare_images(source_batch);
+}
```

Comment:
You don't need to override a method just to call the parent class's version. Just omit this definition and the override declaration above.

You only need to override methods whose behavior you want to change